### ISSUE TITLE
Plugins

### ISSUE THREAD

**Issue by vwidmer** [cybuzuma/aur_opencpn#1]
Have you been able to build any of the plugins for this build? I seem to keep getting these errors.

CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find wxWidgets (missing: wxWidgets_LIBRARIES
  wxWidgets_INCLUDE_DIRS)
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake/Modules/FindwxWidgets.cmake:984 (find_package_handle_standard_args)
  cmake/PluginConfigure.cmake:546 (find_package)
  CMakeLists.txt:153 (include)

**Comment by cybuzuma** [comment]
Yes, I have so far successfully compiled and used the weather-routing and o-charts plugins. I am in the process of fixing up the PKGBUILDs so they can be distributed, so you need to give me a little more time.

Which plugins would be most of interest to you?

**Comment by cybuzuma** [comment]
https://aur.archlinux.org/packages/opencpn-plugin-o_charts is released.
